open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/entities/terms/SubjectAreaTerm.java

```diff
@@ -2,9 +2,18 @@
 /* Copyright Contributors to the ODPi Egeria project. */
 package org.odpi.openmetadata.accessservices.subjectarea.client.entities.terms;
 
+import org.odpi.openmetadata.accessservices.subjectarea.client.SubjectAreaClient;
 import org.odpi.openmetadata.accessservices.subjectarea.client.SubjectAreaEntityClient;
 import org.odpi.openmetadata.accessservices.subjectarea.properties.objects.term.Term;
 
+/**
+ * The SubjectAreaDefinition Open Metadata Access Service (OMAS) API for terms.
```

Comment:
Missing the equivalent javadoc for Category

Reply:
It is not missing. I added this on 8th July.